target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -35,6 +35,12 @@ engenius,eap1300)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "${boardname}:yellow:wlan5g" "phy1tpt"
 	ucidef_set_led_default "mesh" "MESH" "${boardname}:blue:mesh" "0"
 	;;
+engenius,ens620ext)
+        ucidef_set_led_wlan "wlan2g" "WLAN2G" "${boardname}:green:wlan2G" "phy0tpt"
+        ucidef_set_led_wlan "wlan5g" "WLAN5G" "${boardname}:green:wlan5G" "phy1tpt"
+        ucidef_set_led_netdev "lan" "LAN" "${boardname}:green:lan" "eth0"
+        ucidef_set_led_netdev "zlan1" "ZLAN1" "${boardname}:green:zlan1" "eth1"
```

Comment:
I'm curious, do you know what's up with the "ZLAN1" name? Is this something from the stock firmware or where is it coming from?

Reply:
(I've been been feeling my way into adding device support) Device has two lan ports and two LEDs. Was not sure if "lan" label had a special meaning so copied that for eth0, adding something else for eth1/lan1.

What is the appropriate/expected change here? lan0/LAN0/green:lan0 eth0 and lan1/LAN1/green/lan1 eth1?

Reply:
I'm sorry, I haven't noticed this earlier.

> What is the appropriate/expected change here?

As a general rule for most labels: follow the label/pictogram/sticker that written/printed/etched on the casing, silk-screened on the PCB or specified in the manual.

As for LED labels: They should follow the linux' kernels upstream recommendation of
"devicename:color:function" as per:
https://github.com/spotify/linux/blob/master/Documentation/leds-class.txt
(See section LED Device Naming)

Reply:
Fixed in next commit.